CONTRIBUTING.md

```diff
@@ -1,61 +1,93 @@
+```
+
+## Making a Pull Request
 
-### Questions
-If you have any question please use one of the following methods:
+Before making a pull request, please discuss your ideas first.
 
-* [ask a question on stackoverflow](https://stackoverflow.com/questions/tagged/kadena)
-* [join our discord](http://discord.io/kadena)
+Make sure to generate the changelog before it gets merged:
 
-> **note** please do not use our issue board for asking a question 
+```bash
+rush change
+```
 
-## Code of Conduct 
-tbd.
+## Conventions
 
-### Conventions
-tbd.
+This repository uses a combination of TypeScript, ESLint and Prettier to adhere
+to coding standards. We try to automate and auto-fix as much as possible using
+the following commands:
 
-## Issues
-This section will help you create issues or filing bugs for Kadena.js. Following these steps will 
-help maintainers and community members solving the issues finding duplicates etc. etc.
+```bash
+rush build   # Compile & build all packages (using TypeScript)
+rush lint    # Lint (and fix) all packages (using ESLint)
+```
 
-1. Before creating an issue  make sure you are running the latest version
-2. Check if the issue doesn't already exist under [issues](https://github.com/kadena-community/kadena.js/issues)
-3. Use a clear and descriptive title for the identification of the problems
-4. Describe in a clear flow how to reproduce the problem. If possible create a gif or use screenshots. 
-5. Include details about the configuration and environment
+Use `rushx` to do the same for only the current package.
 
+For everything else, please discuss.
 
-## Pull Request
-When you want to pick up an issues or want to improve the code it is done via a pull request.
+## Publishing Packages
 
-When filling out the pull request please take note of the following checklists:
-* Link the PR to an issue (if this is applicable).
-* Enable the checkbox to allow maintainer edits so the branch can be updated for a merge. Once you submit your PR, a team member will review your proposal. We may ask questions or request for additional information.
-* As you update your PR and apply changes, mark each conversation as resolved.
+To publish a new version of updated packages, please make sure you:
 
-When your PR is merged!
+- are part of the `@kadena` npm organization
+- have push rights to this repository's `master` branch
+- are on a clean `master` branch
 
-🚀🚀 Thank you 🚀🚀.
+Follow these steps to publish the updated packages:
 
-Now that you are part of the Kadena.js community, see how else you can contribute to the project.
+- Build and test from root
+- Bump the version
+- Publish updated packages
 
+```bash
+rush build
+rush test
+rush version --bump -b master --ignore-git-hooks
+rush publish --apply --publish --add-commit-details --set-access-level public --target-branch master
+rush publish --apply --publish --include-all --add-commit-details --set-access-level public --target-branch master
```

Comment:
Both `rush publish` commands are in the previous version. The difference is `--include-all`. I guess we need only the latter?